.changeset/slimy-turkeys-return.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-permission-node': minor
```

Comment:
This'll likely not be a breaking change right? `patch` in that case

Reply:
+1, great choice on making the API that way - this unblocks the changes we need to make to the plugin author docs too